src/pages/proposal.tsx

```diff
@@ -381,8 +392,17 @@ export default function ProposalPage() {
         image="https://decentraland.org/images/decentraland.png"
       />
       <Navigation activeTab={NavigationTab.Proposals} />
+      <NotMobile>
+        {renderFloatingBar && (
```

Comment:
I'd change this to `renderFloatingBar && proposal` so proposal is not null in FloatingBar and you could remove the condition inside FloatingBar.